docs/source/platform/federation.mdx

```diff
@@ -235,16 +237,16 @@ The managed configuration for the Apollo gateway is exposed through [Google Clou
 
 With [managed federation](#managed-configuration), you have the ability to control which version of your graph a fleet of gateways are running with. For the majority of deployments, rolling over all of your gateways to a new schema version is a good strategy, since changes should be checked to be backwards compatible using [schema validation](/platform/schema-validation/). However, changes at the gateway level may involve a variety of different updates, like transferring type ownership from one service to another. In the case that your infrastructure requires more advanced deployment strategies, we recommend using [graph variants](/platform/schema-registry/#registering-schemas-to-a-variant) to manage different fleets of gateways running with different configurations.
 
-For instance, in order to have a canary deployment, you might maintain two production graphs in the graph manager, one called `prod` and one called `prod-canary`. Your deployment of a change to some federated service named "foo" might look something like this:
+For instance, in order to have a canary deployment, you might maintain two production graphs in the graph manager, one called `prod` and one called `prod-canary`. Your deployment of a change to some federated service named "launches" might look something like this:
 
-1. Check changes in "foo" against `prod` and `prod-canary`:
+1. Check changes in "launches" against `prod` and `prod-canary`:
    ```
-   apollo service:check --tag=prod --serviceName=foo && apollo service:check --tag=prod-canary --serviceName=foo
+   apollo service:check --tag=prod --serviceName=launches && apollo service:check --tag=prod-canary --serviceName=launches
```

Comment:
I know you didn't introduce this, but while we're changing this line: it's _quite_ wide on the deployed version:

![image](https://user-images.githubusercontent.com/841294/62639098-7f3c8500-b947-11e9-8f07-e754b858c676.png)


Suggestion:

```suggestion
   apollo service:check --tag=prod --serviceName=launches
   apollo service:check --tag=prod-canary --serviceName=launches
```